SetupGuide_Plugin_Admin.php

```diff
@@ -78,7 +78,7 @@ public function load() {
 	 * @since 2.0.0
 	 */
 	public function skip() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
Remove the phpcs comment to have it validate the change.

Reply:
Removed PHPCS disable comment.